setup.py

```diff
@@ -9,6 +9,11 @@
 import sys
 from setuptools import setup
 
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = OSError
```

Comment:
This is probably overkill – on Python 3 `FileNotFoundError` is a subclass of `OSError` and we probably need to catch that as well.